content/tracking/ua_ga4_migration.md

```diff
@@ -248,7 +248,7 @@ This method corresponds to the equivalent Enhanced Ecommerce [checkout event](ht
 window.snowplow("trackEnhancedEcommerceCheckoutStep:{trackerName}",  {{dataLayer.ecommerce reference}}, checkoutOption);
 ```
 
-- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/trenhancedecommerce/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step/jsonschema), except `step` which is retrieved from the dataLayer directly.
+- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/tree/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step), except `step` which is retrieved from the dataLayer directly.
```

Comment:
Not sure if it's intentional that you link to the folder or not, just thought it might be better to link the file directly.

```suggestion
- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step/jsonschema/1-0-0), except `step` which is retrieved from the dataLayer directly.
```